go.mod

```diff
@@ -209,3 +210,5 @@ require (
 	mvdan.cc/gofumpt v0.7.0 // indirect
 	mvdan.cc/unparam v0.0.0-20250301125049-0df0534333a4 // indirect
 )
+
+tool golang.org/x/tools/gopls/internal/analysis/modernize/cmd/modernize
```

Comment:
Should we create a `tools` directory and module to include our tooling and keep the top level separate? We can move golangci-lint there too

Reply:
https://github.com/golang/go/issues/73279 describes that modernize shouldn't be contained as dependency, then they recommended to just run by using go run. So I use tool directive for golangci-lint not modernize.